### PR TITLE
Move MapDB storage implementation into quasar

### PR DESCRIPTION
**Pull request by jsantos17**
[ch7966]

### CODE REVIEW

**Comment by jsantos17** [comment]
Published as `130.0.15-dcc6d15`

File: run/src/test/scala/quasar/run/store/StoreCodecSpec.scala [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+  val jsonGen: Gen[Json] =
+    Gen.oneOf(
+      List(
+        Json.obj("foo" -> Json.jString("bar")),
+        Json.jString("quux"),
+        Json.jNumber(42L),
+        Json.array(Json.jString("one"), Json.jString("two"))))
+
+  def modifyGen: Gen[DestinationRef[Json] => DestinationRef[Json]] =
```
Comment (cryogenian) [inline run/src/test/scala/quasar/run/store/StoreCodecSpec.scala]: I can't find if it's used anywhere.

File: run/src/main/scala/quasar/run/store/Store.scala [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+import shims._
+
+object Store {
+  val MapDBTableName = "default"
+
+  def codecStore[A: Show, F[_]: Sync: ContextShift: MonadError_[?[_], StoreError]](
+    path: Path,
+    codec: CodecJson[A],
+    pool: BlockingContext)
+      : Resource[F, IndexedStore[F, UUID, A]] =
```
Comment (cryogenian) [inline run/src/main/scala/quasar/run/store/Store.scala]: Shouldn't this be 4 spaces, not 6?

File: run/src/test/scala/quasar/run/store/StoreCodecSpec.scala (hunk 1 above)
Comment (jsantos17) [reply run/src/test/scala/quasar/run/store/StoreCodecSpec.scala]: Oh right. This was used when I tested this is a prism. Will delete.

File: run/src/main/scala/quasar/run/store/Store.scala (hunk 2 above)
Comment (jsantos17) [reply run/src/main/scala/quasar/run/store/Store.scala]: Right. `emacs` likes six for some reason.

**Comment by jsantos17** [comment]
@cryogenian this should be ready for another look.

**Pull request merged**